muxpi/dut.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "Setting up DUT" 
+stm_armv7 -clr
+stm_armv7 -print "DUT"
+stm_armv7 -dut
+echo "This script contains a lot of sleep commands as it needs to wait for the screens. Patience is key!"
```

Comment:
What is actually the reason for the large amount of sleeps? I have been taught that a good script doesn't have any sleeps. Is there no other way to wait for commands to finish? Expecting output or a return code or something?

Reply:
The 30s one waits for the rpi to boot. Since there are two screens running simultaneously, they need to wait for one and other. Im not sure about expecting output or a return since the screens are running on a serial device.